src/components/dev-hub/feedback/feedback-container.js

```diff
@@ -11,21 +24,83 @@ const feedbackItems = graphql`
     }
 `;
 
-const FeedbackContainer = () => {
+const StyledButtonContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 4px;
+`;
+
+const StyledForm = styled('form')`
+    padding: 0 ${size.medium};
+
+    button,
+    > label,
+    ${StyledInputContainer} {
+        margin: ${size.default} 0;
+    }
+`;
+
+const StyledDescription = styled(P)`
+    display: inline-block;
+    font-family: Akzidenz;
+    font-size: ${fontSize.small};
+    line-height: ${lineHeight.small};
+`;
+
+const MODAL_WIDTH = '400px';
+
+const FeedbackContainer = ({ onSubmit, starRatingFlow }) => {
+    const [step, setStep] = useState(0);
     const data = useStaticQuery(feedbackItems);
-    const {
-        OneStarRatingFlow,
-        TwoStarRatingFlow,
-        ThreeStarRatingFlow,
-        FourStarRatingFlow,
-        FiveStarRatingFlow,
-    } = dlv(data, 'strapiFeedbackRatingFlow', {});
-    // Uncomment below to see shape of data, other forms
-    // console.log(data);
-    // TODO: Implement feedback UI
-    return (
-        <CMSForm form={OneStarRatingFlow ? OneStarRatingFlow.forms[1] : []} />
+
+    const { ratingFlow = {}, stepsCounter } = getStarRatingFlow(
+        data,
+        starRatingFlow,
+        step
     );
+
+    const { title, description, cta: button } = ratingFlow;
+
+    const isActiveModal = step < stepsCounter;
+
+    return isActiveModal ? (
+        <Modal
+            isOpenToStart={true}
+            verticallyCenter
+            contentStyle={{
+                'max-width': MODAL_WIDTH,
+                'min-width': MODAL_WIDTH,
```

Comment:
```suggestion
                maxWidth: MODAL_WIDTH,
                minWidth: MODAL_WIDTH,
```

Apologies as I missed this before but I saw an error in the console on using kebab-case vs camelCase. Let's use the camelCasing here please.